File: src/test/java/org/mtransit/parser/TimeUtilsTest.java
package org.mtransit.parser;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class TimeUtilsTest {

	@Test
	public void testConvertTimeToString_0() {
		// Arrange
		// Act
		String result = TimeUtils.convertTimeToString(0);
		// Assert
		assertEquals("000000", result);
	}

	@Test
	public void testConvertTimeToString_12345() {
		// Arrange
		// Act
		String result = TimeUtils.convertTimeToString(12345);
		// Assert
		assertEquals("012345", result);
	}

	@Test
	public void testConvertTimeToString_123456() {
		// Arrange
		// Act
		String result = TimeUtils.convertTimeToString(123456);
		// Assert
		assertEquals("123456", result);
	}
}

File: src/test/java/org/mtransit/parser/TimeUtilsTest.kt
package org.mtransit.parser

import org.junit.Assert
import org.junit.Test

class TimeUtilsTest {

    @Test
    fun testConvertTimeToString_0() {
        // Arrange
        // Act
        val result = TimeUtils.convertTimeToString(0)
        // Assert
        Assert.assertEquals("000000", result)
    }

    @Test
    fun testConvertTimeToString_12345() {
        // Arrange
        // Act
        val result = TimeUtils.convertTimeToString(12345)
        // Assert
        Assert.assertEquals("012345", result)
    }

    @Test
    fun testConvertTimeToString_123456() {
        // Arrange
        // Act
        val result = TimeUtils.convertTimeToString(123456)
        // Assert
        Assert.assertEquals("123456", result)
    }
}